modules/cluster-logging-elasticsearch-tolerations.adoc

```diff
@@ -1,66 +1,82 @@
 // Module included in the following assemblies:
 //
-// * logging/cluster-logging-elasticsearch.adoc
+// * logging/scheduling_resources/logging-taints-tolerations.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-elasticsearch-tolerations_{context}"]
 = Using tolerations to control the log store pod placement
 
-You can control which nodes the log store pods runs on and prevent
-other workloads from using those nodes by using tolerations on the pods.
-
-You apply tolerations to the log store pods through the `ClusterLogging` custom resource (CR)
-and apply taints to a node through the node specification. A taint on a node is a `key:value pair` that
-instructs the node to repel all pods that do not tolerate the taint. Using a specific `key:value` pair
-that is not on other pods ensures only the log store pods can run on that node.
-
-By default, the log store pods have the following toleration:
+By default, the log store pods have the following toleration configuration:
```

Comment:
Thed efault tolerations to logStore is not added by CLO .   It is added by  cluster scheduler into the pod sources.

For example:
oc get pods elasticsearch-cdm-5oupch6e-1-6978fd6859-ml9bl -o yaml
```
apiVersion: v1
kind: Pod
metadata:
  name: elasticsearch-cdm-5oupch6e-1-6978fd6859-ml9bl
  namespace: openshift-logging
spec:
  nodeSelector:
    kubernetes.io/os: linux
  serviceAccount: elasticsearch
  serviceAccountName: elasticsearch
  tolerations:
  - effect: NoSchedule
    key: node.kubernetes.io/disk-pressure
    operator: Exists
  - effect: NoExecute
    key: node.kubernetes.io/not-ready
    operator: Exists
    tolerationSeconds: 300
  - effect: NoExecute
    key: node.kubernetes.io/unreachable
    operator: Exists
    tolerationSeconds: 300
  - effect: NoSchedule
    key: node.kubernetes.io/memory-pressure
    operator: Exists

```

Reply:
Also tried updating this for the collector example